packages/ui-settings/src/types.ts

```diff
@@ -30,18 +30,9 @@ export interface NetworkSpecsStruct {
   unit: string;
 }
 
-export type Endpoint = JsonRpcEndpoint | SubstrateConnectEndpoint;
-export interface JsonRpcEndpoint {
-  type: EndpointType.jrpc;
-  url: string;
+export type Endpoint = {
+  type: EndpointType;
+  param: string;
```

Comment:
```suggestion
export interface Endpoint {
  type: EndpointType;
  param: string;
```

url was dropped from the previous version, I believe that is why CI is complaining bitterly.